health-services/project/src/main/resources/application.properties

```diff
@@ -162,8 +164,16 @@ project.resource.consumer.bulk.update.topic=update-project-resource-bulk-topic
 project.resource.consumer.bulk.delete.topic=delete-project-resource-bulk-topic
 
 project.mdms.module=HCM-PROJECT-TYPES
+task.mdms.module=HCM-TASK-QUANTITY-VALIDATION
 egov.location.hierarchy.type=ADMIN
 
 #---------Attendance-----------#
 project.staff.attendance.topic=project-staff-attendance-health-topic
 
+
+project.task.resource.quantity.pattern=^(?!0\\.0$)(0\\.5|([1-9]\\d*|0)\\.0|[1-9]\\d*\\.5)$
+
```

Comment:
is this needed after mdms backed validation?

Reply:
Removed error message and regex patterns from application.properties